test/xcm/[V3]RU.moonriver.transfer.test.ts

```diff
@@ -32,7 +32,7 @@ const alith = keyring.addFromUri(
   "ethereum",
 );
 
-describe("[V3][V3] XCM tests for Mangata <-> moonriver", () => {
+describe.skip("[V3][V3] XCM tests for Mangata <-> moonriver", () => {
```

Comment:
if this is the only describe in the file, plz remove the group too. ow, the test setup will fail,

Reply:
Done